src/components/ListItemDetails.jsx

```diff
@@ -0,0 +1,21 @@
+import './ListItem.css';
+
+export function ListItemDetails({
+	totalPurchases,
+	lastPurchasedDate,
+	nextPurchasedDate,
+}) {
+	return (
+		<section>
+			{lastPurchasedDate ? (
+				<>
+					<p>You have purchased this item {totalPurchases} times</p>
```

Comment:
I think you could change this line to `<p>You have purchased this item {totalPurchases} time{totalPurchases > 1 ? 's' : ''}</p>` for the instance where a user has purchased the item once.